src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -54,8 +61,22 @@ private JsonObject extendedLoan(JsonObject loan, Item item) {
 
     return loan;
   }
+
+  private void additionalLoanPolicyProperties(JsonObject loanRepresentation, LoanPolicy loanPolicy) {
+    if (loanPolicy == null) {
+      log.info("Unable to add loan policy properties to loan {}," + " loanPolicy is null", loanRepresentation.getString("id"));
+      return;
+    }
+    JsonObject loanPolicySummary = loanRepresentation.containsKey(LoanProperties.LOAN_POLICY)
+        ? loanRepresentation.getJsonObject(LoanProperties.LOAN_POLICY)
+        : new JsonObject();
+
+    loanPolicySummary.put("loanPolicyName", loanPolicy.getName());
```

Comment:
Given this is being added here, why is the internal representation of the loan also updated?

Reply:
Sure, I removed it from the Loan class